app/logic/displayName.py

```diff
@@ -0,0 +1,17 @@
+from app.models import * 
```

Comment:
This function doesn't need it's own file. It is course related so should probably go with other course logic.